packages/web/pages/dashboard.tsx

```diff
@@ -0,0 +1,258 @@
+        layouts={gridLayouts}
+        breakpoints={{ xl: 1920, lg: 1180, md: 900, sm: 768, xs: 480, xxs: 0 }}
+        preventCollision={false}
+        cols={{ xl: 12, lg: 12, md: 12, sm: 4, xs: 4, xxs: 4 }}
+        rowHeight={135}
+        autoSize
+        // isBounded
+        isDraggable={!!editable}
+        isResizable={!!editable}
+        onDragStart={toggleScrollLock}
+        onDragStop={toggleScrollLock}
+        onResizeStart={toggleScrollLock}
+        onResizeStop={toggleScrollLock}
+        transformScale={1}
+        margin={{
+          xl: [20, 20],
+          lg: [20, 20],
+          md: [20, 20],
+          sm: [20, 20],
+          xs: [20, 20],
+          xxs: [20, 20],
+        }}
+        containerPadding={{
+          xl: [20, 20],
+          lg: [20, 20],
+          md: [15, 15],
+          sm: [15, 15],
+          xs: [10, 10],
+          xxs: [10, 10],
+        }}
+      >
+        {/* <DashboardSection key="latest" id="latest" containerQuery={queryData}> */}
+        <Box key="latest" className="gridItem">
+          <GridItem title="Latest Content" sx={gridConfig.latest}>
+            <LatestContent />
+          </GridItem>
+        </Box>
+        <Box key="xp" className="gridItem">
+          <GridItem title="XP" sx={gridConfig.xp}>
+            <XP />
+          </GridItem>
+        </Box>
+        <Box key="seed" className="gridItem">
+          <GridItem title="Seed" sx={gridConfig.seed}>
+            <Seed />
+          </GridItem>
+        </Box>
+        <Box key="calendar" className="gridItem">
+          <GridItem title="Calendar" sx={gridConfig.calendar}>
+            <Calendar />
+          </GridItem>
+        </Box>
+        <Box key="leaderboard" className="gridItem">
+          <GridItem title="Leaderboard" sx={gridConfig.leaderboard}>
+            <Leaderboard />
+          </GridItem>
+        </Box>
+      </ResponsiveGridLayout>
+    </Box>
+  );
+};
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function resetLayouts() {
+  if (getFromLS('metagame-dashboard') === null) {
+    return false;
+  }
+  return global.localStorage.removeItem('metagame-dashboard');
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function getFromLS(key: string) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let ls: any = {};
+  if (global.localStorage) {
+    try {
+      const dashboard = global.localStorage.getItem('metagame-dashboard');
+      ls = dashboard !== null ? JSON.parse(dashboard) : {};
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.log('getFromLS error: ', e);
+      // return null;
+    }
+  }
+  return ls[key];
+}
+
+// eslint-disable-next-line
+export function saveToLS(key: string, value: any) {
+  if (global.localStorage) {
+    global.localStorage.setItem(
+      'metagame-dashboard',
+      JSON.stringify({
+        [key]: value,
+      }),
+    );
```

Comment:
Multiple calls to this will overwrite the value each time.

A more robust implementation would be to default `key` to `null` on `getFromLS` and if `key` is `null` return the whole settings object. This would allow you to check if the object is set above, and you could do something here like:

```suggestion
    const settings = getFromLS();
    settings[key] = value;
    global.localStorage.setItem(
      'metagame-dashboard',
      JSON.stringify(settings),
    );
```

Reply:
as we stated in the main pr of feature/dashboard, the localstorage implementation is just temporary, until we implement saving of the layout to the server, but thank you for the suggestion.